### PR TITLE
configurable number of parallel tasks in deps compilation

### PR DESCRIPTION
**Pull request by mm-s**
* fixed error message when attempting to create an existing _build dir.
* I needed to set the -j flag from console
* Takes default value of -j from /proc/cpuinfo
* rewrite of help function

### CODE REVIEW

File: scripts/configure-manual.sh [hunk 1]
```diff
@@ -276,12 +284,24 @@ function make_build_dir {
 #-------------------------------------------------------
 
 prog=$0
-jobs=8
+jobs=`cat /proc/cpuinfo | grep "^processor" | wc -l`
```
Comment (Jaguar0625) [inline scripts/configure-manual.sh:287]: not sure of your intent, but this won't work on `darwin`

Comment (mm-s) [reply scripts/configure-manual.sh:287]: I didn't consider darwin ..., I was solving my own problem here.  I'll update

File: scripts/configure-manual.sh [hunk 2]
```diff
@@ -284,7 +284,10 @@ function make_build_dir {
 #-------------------------------------------------------
 
 prog=$0
-jobs=`cat /proc/cpuinfo | grep "^processor" | wc -l`
+jobs=4
+if [[ -f /proc/cpuinfo ]]; then
+    jobs=`cat /proc/cpuinfo | grep "^processor" | wc -l`
```
Comment (Jaguar0625) [inline scripts/configure-manual.sh:289]: spaces => tab

Comment (mm-s) [reply scripts/configure-manual.sh:289]: oh, ok

Comment (mm-s) [reply scripts/configure-manual.sh]: updated and rebased

**Review by Jaguar0625 — APPROVED** [review]

**Pull request merged**